extensions/amp-script/0.1/test/validator-amp-script-not-origin-trial.out

```diff
@@ -34,4 +34,4 @@ FAIL
 |    <amp-script layout=container src="https://example.com/foo.js"><div class="root"><p>Some text to hydrate.</p></div></amp-script>
 |  </body>
 >>       ^~~~~~~~~
-amp-script/0.1/test/validator-amp-script-not-origin-trial.html:34:6 The tag 'amp-experiment-token' is missing or incorrect, but required by 'amp-script'. (see https://amp.dev/documentation/components/amp-script) [AMP_TAG_PROBLEM]
+amp-script/0.1/test/validator-amp-script-not-origin-trial.html:34:6 The tag 'amp-experiment-token' is missing or incorrect, but required by 'amp-script'. (see https://amp.dev/documentation/components/amp-script) [AMP_TAG_PROBLEM]
```

Comment:
Intentional change?

Reply:
Hmm this was auto-generated by `gulp validator --update_tests`. I think it's harmless.